config/test_config.py

```diff
@@ -72,3 +72,10 @@ def test_configure_logger(self):
 
         with logger.catch():
             raise ValueError
+
+    def test_configure_openai(self, monkeypatch):

```

Comment:
Since this is a positive test, you can just combine it with the one on line 46-53. This test can be made into a negative test instead where you patch the env to be empty to validate the error handling of the config

Reply:
On line 46, before the AppConfig setup, since what we want to test is only the required property of an `OPENAI_API_KEY`, you can just patch it on line 46

Reply:
Alternatively, you can setup a fixture to monkeypatch the env var with `test-openai-api-key` and include it in the `test_config` and `test_config_default_port` function